library/src/main/java/com/getstream/sdk/chat/viewmodel/channels/ChannelListViewModelBinding.kt

```diff
@@ -13,8 +13,12 @@ fun ChannelsViewModel.bindView(view: ChannelsView, lifecycle: LifecycleOwner) {
                 view.hideLoadingView()
             }
             is ChannelsViewModel.State.Loading -> {
-                view.hideEmptyStateView()
-                view.showLoadingView()
+                if (it.isLoading) {
+                    view.hideEmptyStateView()
+                    view.showLoadingView()
+                } else {
```

Comment:
Why do we have a `loading` state that is "Not loading"? :thinking: 
It seems a bit confusing
Maybe we could have an extra state if it is needed and split this logic

Reply:
Loading event comes from livedata. `QueryChannelsControllerImpl.loading: LiveData<Boolean>` emmits true/false.

Reply:
`QueryChannelsControllerImpl.loading: LiveData<Boolean>` is redundant for the need of app here. Simplified.